config/environments/development.rb

```diff
@@ -1,28 +1,41 @@
+require "active_support/core_ext/integer/time"
+
 Rails.application.configure do
   config.cache_classes = false
   config.eager_load = false
-  config.consider_all_requests_local       = true
+  config.consider_all_requests_local = true
+
+  # Tell Rails not to send server timing headers
+  config.server_timing = false
 
   # Enable/disable caching. By default caching is disabled.
   # Run rails dev:cache to toggle caching.
-  if Rails.root.join('tmp', 'caching-dev.txt').exist?
+  if Rails.root.join("tmp/caching-dev.txt").exist?
     config.action_controller.perform_caching = true
     config.action_controller.enable_fragment_cache_logging = true
+
     config.cache_store = :memory_store
     config.public_file_server.headers = {
-      'Cache-Control' => "public, max-age=#{2.days.to_i}"
+      "Cache-Control" => "public, max-age=#{2.days.to_i}"
     }
   else
     config.action_controller.perform_caching = false
     config.cache_store = :null_store
   end
 
   config.active_support.deprecation = :raise
+  # Raise exceptions for disallowed deprecations.
+  config.active_support.disallowed_deprecation = :raise
+  # Tell Active Support which deprecation messages to disallow.
+  config.active_support.disallowed_deprecation_warnings = []
+
   config.active_record.migration_error = :page_load
+  config.active_record.verbose_query_logs = true
+
   config.assets.debug = true
   config.assets.digest = true
+  config.assets.quiet = true
```

Comment:
This was `true` before (see line 26 below) but I put all the `config.assets` lines together.